ui/src/app/views/homepage/component.tsx

```diff
@@ -1,201 +1,33 @@
 import {Component} from '@angular/core';
 import {navigate, queryParamsStore} from 'app/utils/navigation';
-import {environment} from 'environments/environment';
 
 import * as fp from 'lodash/fp';
 import * as React from 'react';
 
-import {AlertClose, AlertWarning} from 'app/components/alert';
 import {
   Button,
   CardButton,
   Clickable,
-  styles as buttonStyles,
 } from 'app/components/buttons';
 import {ClrIcon} from 'app/components/icons';
 import {Modal, ModalFooter} from 'app/components/modals';
 import {TooltipTrigger} from 'app/components/popups';
 import {Spinner} from 'app/components/spinners';
 import {configApi, profileApi} from 'app/services/swagger-fetch-clients';
-import {hasRegisteredAccessFetch, reactStyles, ReactWrapperBase, withStyle, withUserProfile} from 'app/utils';
+import {hasRegisteredAccessFetch, reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils';
 import {QuickTourReact} from 'app/views/quick-tour-modal/component';
 import {RecentWork} from 'app/views/recent-work/component';
+import {RegistrationPage} from 'app/views/registration-page/component';
 import {
   BillingProjectStatus,
   Profile,
 } from 'generated/fetch';
 
-
-const styles = reactStyles({
+export const homepageStyles = reactStyles({
```

Comment:
Is there a reason to change this from `styles` to `homepageStyles`? It breaks our convention of page independent styles being called `styles`.

Reply:
👍